packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -112,6 +112,12 @@ class ImagePicker {
   /// image types such as JPEG and on Android PNG and WebP, too. If compression is not
   /// supported for the image that is picked, a warning message will be logged.
   ///
+  /// The `limit` parameter modifies the maximum number of images that can be selected.
+  /// On Android, [ImagePickerAndroid.useAndroidPhotoPicker] must be set to `true`
+  /// to use the `limit` functionality. And it can only be ensured to take effect
+  /// on Android 13 or above. Otherwise, it depends on whether the corresponding
+  /// system app supports it.
```

Comment:
We do not generally get this specific about platform-specific details in the app-facing package:
- They tend to get outdated because they are about details that are controlled in another package.
- They don't scale well to many platforms (e.g., you haven't discussed desktop here, and specifically calling out only Android makes it seem like desktop should be fine, which isn't the case).
- They can't scale to third-party implementations.

I would much prefer we just use [the same comment that we use for the platform interface layer](https://github.com/flutter/packages/blob/822d8852e7003e2366458873b4ecac23ce7f812b/packages/image_picker/image_picker_platform_interface/lib/src/types/multi_image_picker_options.dart#L31).

This level of platform-specific detail is something we could put in the Android package's README.

Reply:
Got it.